tools/templates/documentation.json

```diff
@@ -6990,6 +6990,176 @@
 					"protocolProfileBehavior": {},
 					"_postman_isSubFolder": true
 				},
+				{
+					"name": "CRL",
+					"item": [
+						{
+							"name": "Read System CRLs",
+							"protocolProfileBehavior": {
+								"disableBodyPruning": true
+							},
+							"request": {
+								"method": "GET",
+								"header": [],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \n}",
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "https://{{$hostname}}/api/v1/system/crl",
+									"protocol": "https",
+									"host": [
+										"{{$hostname}}"
+									],
+									"path": [
+										"api",
+										"v1",
+										"system",
+										"crl"
+									]
+								},
+								"description": "Read installed CRLs.<br><br>\n\n_Requires at least one of the following privileges:_ [`page-all`, `page-system-camanager`]"
+							},
+							"response": []
+						},
+						{
+							"name": "Create System CRL",
+							"request": {
+								"method": "POST",
+								"header": [],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n\t\"method\": \"internal\",\n\t\"descr\": \"TEST CA\",\n\t\"caref\": \"61c410f04b782\",\n\t\"lifetime\": 3650,\n\t\"serial\": 10\n}",
```

Comment:
Small consistency issue here, it is creating a CRL with the description of `TEST CA` which may be a little confusing. Not a huge issue, I've definitely left bigger inconsistencies. 😄

Reply:
Nice catch. fixed